errors/either/either.sc

```diff
@@ -1,18 +1,28 @@
+/**
+ * Creating an either
+ */
 val either1: Either[String,Int] = Right(1)
 val either2: Either[String,Int] = Left("error!")
 
-// map
+
+/**
+ * Transforming the either
+ */
+// map - transform the value inside the either, if it exists
 val doubled: Either[String,Int] = either1.map(n => n*2)
 
-// flatMap
+// flatMap - transform the value inside, with a function that returns an either
```

Comment:
inside the right

Reply:
although on that latter point, maybe it breaks the parity with `Option` & `Try` which is quite nice

Reply:
yeah, generally you handle the Left only at the end when you 'escape' the Either